examples/flipper/lib.rs

```diff
@@ -51,4 +51,74 @@ pub mod flipper {
             assert!(flipper.get());
         }
     }
+
+    #[cfg(all(test, feature = "e2e-tests"))]
+    mod e2e_tests {
+        use super::*;
+        use ink_e2e::build_message;
+
+        type E2EResult<T> = std::result::Result<T, Box<dyn std::error::Error>>;
+
+        #[ink_e2e::test]
+        async fn flip(mut client: ink_e2e::Client<C, E>) -> E2EResult<()> {
```

Comment:
nit: I would change these to be `it_works` and `default_works` just to match the existing test names (or the other way around, as long as they're consistent)